<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCapitalChangeSearchDialog</class>
 <widget class="QDialog" name="frmCapitalChangeSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>资金变动流水查询</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1021</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>881</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_st">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>结束日期</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_ed">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_query">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
        font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>开始日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>29</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>存取方向</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_change">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_first">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>310</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>|&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
        font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_num">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #2a3a57;
color:#ffffff;
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_page">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #2a3a57;
color:#ffffff;
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pre">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
        font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_last">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
        font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_output_excel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>35</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
#pushButton_output_excel{
color:white;
        font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
#pushButton_output_excel:enabled:hover{
        background: #227dc3;
}

#pushButton_output_excel:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>输出到excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_option">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
        font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>设置显示列</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
